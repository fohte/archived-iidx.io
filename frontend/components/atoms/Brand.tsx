const Brand: React.SFC = () => <span>iidx.io</span>

export default Brand
