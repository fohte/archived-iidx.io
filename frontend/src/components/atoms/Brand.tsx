import * as React from 'react'

const Brand: React.SFC = () => <span>iidx.io</span>

export default Brand
