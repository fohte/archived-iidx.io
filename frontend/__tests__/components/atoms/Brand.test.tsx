import { shallow } from 'enzyme'
import * as React from 'react'
import * as renderer from 'react-test-renderer'

import Brand from '@app/components/atoms/Brand'

describe('Brand', () => {
  it('renders the brand', () => {
    const brand = shallow(<Brand />)
    expect(brand.text()).toEqual('iidx.io')
  })

  it('renders correctly', () => {
    const tree = renderer.create(<Brand />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
