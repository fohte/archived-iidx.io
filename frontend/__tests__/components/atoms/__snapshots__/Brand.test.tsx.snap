// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Brand renders correctly 1`] = `
<span>
  iidx.io
</span>
`;
